refs/biblio.json

```diff
@@ -3239,67 +3239,71 @@
             }
         }
     },
+
     "dcat-ap": {
-        "href": "https://joinup.ec.europa.eu/asset/dcat_application_profile/",
+        "href": "http://data.europa.eu/w21/ac376c94-74cf-4dd7-ade7-267d6a4ec4dc",
```

Comment:
No https?

Reply:
@andrea-peregocan you please address this comment? Thanks!

Reply:
@andrea-perego can you please address this comment? Thanks!

Reply:
@marcoscaceres , @tobie , these persistent URIs are operated by the Persistent URI Registry of EU Institutions and Bodies (http://data.europa.eu/), which currently uses HTTP. 

I will update them once HTTPS will be supported.

Reply:
Thanks for checking.